tests/README.md

```diff
@@ -25,6 +25,12 @@ To run just `codegen` tests for a single language (replace rust with language of
 cargo test -p wit-bindgen-rust
 ```
 
+To run just `codegen` tests for a single language (replace rust with language of choice: `go`, `c`, `csharp`, etc.) and a single wit file (replace `flags` with whatever wit file should be tested):
```

Comment:
Duplicated in the text below?

Reply:
It is, but this is the specific instruction for running a `codegen` test for a single language and single wit, which wasn't covered explicitly elsewhere.